src/main/java/com/laytonsmith/core/events/drivers/PlayerEvents.java

```diff
@@ -1537,9 +1540,17 @@ public boolean modifyEvent(String key, Construct value, BindableEvent event) {
                     }
                 }
 				if("format".equals(key)){
+					String format = value.nval();
+					if(format == null) {
+						throw new CRENullPointerException("The \"format\" key in " + modify_event.class.getSimpleName() + " for the " + this.getName()
+								+ " event may not be null.", Target.UNKNOWN);
+					} else if(format.replaceAll("%%", "").replaceAll("\\%\\%|\\%[12]\\$s", "").contains("%")) {
```

Comment:
Since the ..Event.setFormat(String format) throws an exception when the format is invalid, I moved the regex to the catch so that it will only be used to generate a better exception and not run on valid use.